packages/components/src/internal/components/entities/EntityInsertPanel.tsx

```diff
@@ -54,12 +54,13 @@ import {
     Location,
     Progress,
     QueryColumn,
-    QueryGridModel,
     queryGridInvalidate,
+    QueryGridModel,
     QueryInfo,
     RemoveEntityButton,
     removeQueryGridModel,
     resolveErrorMessage,
+    SampleCreationTypeModel,
```

Comment:
To match the other nouns in this file, should this be more generally called `EntityCreationTypeModel`?